cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,48 @@
+const { faker } = require('@faker-js/faker');
+
 describe('', () => {
-  before(() => {
+  let user;
+  const article = {
+    title: faker.lorem.word(),
+    about: faker.lorem.words(2),
+    content: faker.lorem.words(10),
+    tag: faker.lorem.word()
+  };
 
+  beforeEach(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
   });
 
-  it('', () => {
+  it('should provide an ability to create an article', () => {
+    cy.login(user.email, user.username, user.password);
+
+    cy.visit('/editor');
+
+    cy.get('[placeholder="Article Title"]').type(article.title);
```

Comment:
I created an article via API. We have a ready command for it in commands. But, I am sure that your code would work too.

Reply:
This is e2e test and we should check UI) We make a requests for setting up the test data. In this test we are using login to setup new user. Other flow will be through UI